pages/c/_cluster/index.vue

```diff
@@ -268,21 +268,28 @@ export default {
 
 <template>
   <section>
-    <header>
-      <h1>
-        <t k="clusterIndexPage.header" :name="cluster.nameDisplay" />
-      </h1>
-      <div class="actions">
-        <button
-          ref="cluster-actions"
-          type="button"
-          class="btn btn-sm role-multi-action actions"
-          aria-haspopup="true"
-          aria-expanded="false"
-          @click="showActions"
-        >
-          <i class="icon icon-actions" />
-        </button>
+    <header class="row">
+      <div class="span-6">
```

Comment:
The title is now limited to half the screen, and a single actions button on the other half?

Reply:
Fixed that, expanded it to 11->1.